Bottom: 1c67274d672319cb18f13cb4dd8f033f8eb744ad
Top:    1282ea50ca5f23f0307e86b9bc16fc1eece3c12a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-08 21:29:24 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
